safe-global/web-core
# fix: use required confirmations when tx pending

## What it solves

Missing confirmation for pending transactions.

## How this PR fixes it

If a transaction is pending an the submitted number of confirmations does not meet the required threshold, the number of confirmations shown is that of the threshold. The confirmations in the transaction stepper is also hidden whilst pending as the confirmations array may not contain the last signer.

## How to test it

Create and immediately execute a transaction. Observe the correct number of signers in the summary and stepper, as well as the confirmees no longer visible in the stepper until the transaction has successfully indexed.

## Screenshots
![confirmations](https://user-images.githubusercontent.com/20442784/186115492-9bfadb19-9df7-4ccd-a7ad-9535f37e63d0.gif)
